src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -148,10 +147,7 @@ export class ReviewRoute extends Component<ReviewProps> {
           order,
           o => o.lineItems?.edges?.[0]?.node?.artwork?.slug
         )
-        return router.push({
-          pathname: `/artwork/${artworkId}`,
-          state: { offerOrderHasBeenSubmitted: true },
-        })
+        return router.push(`/artwork/${artworkId}?orderHasBeenSubmitted=true`)
```

Comment:
Are we using camel-cased params generally? 🤔

Reply:
True!